packages/fuels-test-helpers/src/signers.rs

```diff
@@ -182,4 +197,88 @@ mod tests {
         }
         Ok(())
     }
+
+    #[tokio::test]
+    async fn wallets_with_private_keys() -> Result<(), Box<dyn std::error::Error>> {
+        let num_wallets = 3;
+
+        let private_keys = vec![
+            SecretKey::from_str(
+                "5f70feeff1f229e4a95a7056e8b4d80d0b24b565674860cc213bdb07127ce1b1",
+            )?,
+            SecretKey::from_str(
+                "705ffeeff1f229e4a95e1056e8b4d80d0b24b565674860cc213bdb07127ce1b1",
+            )?,
+        ];
+
+        let config =
+            WalletsConfig::new(Some(num_wallets), None, None).with_private_keys(private_keys);
```

Comment:
Some separation between arrange, act and assert will make this test more readable.